packages/console/src/pages/OrganizationRoleDetails/Permissions/index.tsx

```diff
@@ -199,6 +202,13 @@ function Permissions({ organizationRoleId }: Props) {
           }}
         />
       )}
+      <OrganizationRolePermissionsAssignmentModal
+        organizationRoleId={organizationRoleId}
+        isOpen={isAssignScopesModalOpen}
+        onClose={() => {
+          setIsAssignScopesModalOpen(false);
```

Comment:
Do we need to mutate the data on modal close?

Reply:
The data will be mutated inside the `AssignModal` before the modal is closed.